pkg/operator/k8sutil/prometheus.go

```diff
@@ -63,7 +63,7 @@ func GetServiceMonitor(name string, namespace string, portName string) *monitori
 				{
 					Port:     portName,
 					Path:     "/metrics",
-					Interval: "5s",
+					Interval: "10s",
```

Comment:
We likely need a cluster CR setting to make this configurable. Users may want a different value here.

Reply:
@travisn this configurable by the `interval` from the monitoring section? isn't it?

Reply:
I see, this is just the default, and the `interval` setting already allows setting it to some other value